.github/scripts/check-folders.js

```diff
@@ -4,63 +4,81 @@ const yaml = require('yaml')
 const fs = require('fs')
 const path = require('path')
 
```

Comment:
Small cleanup in this file to make it easier to understand, improvements on the comments welcome.